dhis-2/dhis-services/dhis-service-dxf2/src/main/java/org/hisp/dhis/dxf2/events/enrollment/AbstractEnrollmentService.java

```diff
@@ -689,23 +744,33 @@ public ImportSummary updateEnrollmentForNote( Enrollment enrollment )
     @Override
     public ImportSummary deleteEnrollment( String uid )
     {
-        User user = currentUserService.getCurrentUser();
-        
         ProgramInstance programInstance = programInstanceService.getProgramInstance( uid );
+        String descMsg;
```

Comment:
Set to null or empty string to be explicit.

Reply:
Fixed